src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -3691,9 +3755,31 @@ private void MergePartialMembers(
                 }
             }
 
-            memberNames.Free();
+            foreach (var pair in membersByName)
+            {
+                if (pair.Value is ArrayBuilder<Symbol> arrayBuilder)
+                {
+                    foreach (var symbol in arrayBuilder)
+                    {
+                        fixupNotMergedPartialProperty(symbol);
+                    }
+                }
+                else
+                {
+                    fixupNotMergedPartialProperty((Symbol)pair.Value);
+                }
+            }
 
-            void mergePartialMethods(ref Dictionary<ReadOnlyMemory<char>, ImmutableArray<Symbol>> membersByName, ReadOnlyMemory<char> name, SourceOrdinaryMethodSymbol currentMethod, SourceOrdinaryMethodSymbol prevMethod, BindingDiagnosticBag diagnostics)
+            static void fixupNotMergedPartialProperty(Symbol symbol)
+            {
+                Debug.Assert(symbol.IsPartialMember());
+                if (symbol is SourcePropertySymbol { OtherPartOfPartial: null } property)
+                {
+                    property.SetMergedBackingField(property.DeclaredBackingField);
+                }
+            }
+
+            void mergePartialMethods(ArrayBuilder<Symbol> nonTypeMembers, SourceOrdinaryMethodSymbol currentMethod, SourceOrdinaryMethodSymbol prevMethod, BindingDiagnosticBag diagnostics)
```

Comment:
>mergePartialMethods [](http://example.com/codeflow?start=17&length=19)

static?

Reply:
> static?

I do not think I changed anything that affects ability to place the modifier on the method. Therefore, I won't try adding it, unless there is other feedback to take care.

Reply:
Scratch that, it looks like I removed ```DuplicateMembersByNameIfCached``` call

Reply:
It looks like the containing method `MergePartialMembers` could be marked static too